lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -604,6 +604,9 @@ def connect_invocation_endpoint(endpoint_name, endpoint_suffix, action, conditio
     connect_invocation_endpoint('show', '', action='show_invocation')
     connect_invocation_endpoint('show_report', '/report', action='show_invocation_report')
     connect_invocation_endpoint('show_report_pdf', '/report.pdf', action='show_invocation_report_pdf')
+    connect_invocation_endpoint('export_bco', '/export_bco', action='export_invocation_bco')
+    connect_invocation_endpoint('get_bco', '/get_bco', action='download_invocation_bco')
+    connect_invocation_endpoint('view_bco', '/view_bco', action='view_invocation_bco')
```

Comment:
The last endpoint doesn't seem to exist yet?

Reply:
No. I could use a little advice. 
The `download_invocation_bco` doesn't work to actually produce a download file yet. it outputs a pretty JSON to the browser. 
The last endpoint is what i am working on now thanks to the advice of @guerler on the Galaxy Gitter.